spynnaker/pyNN/models/neuron/synapse_dynamics/abstract_synapse_dynamics.py

```diff
@@ -150,7 +150,7 @@ def get_delay_variance(self, connector, delays, synapse_info):
         """ Get the variance in delay for the synapses
 
         :param AbstractConnector connector:
-        :param ~numpy.ndarray delays:
```

Comment:
Delays param still used so why remove docs?

Reply:
Docs added back in.